pkg/automation/automation.go

```diff
@@ -240,31 +263,12 @@ func installLicenseSecret(clientset *kubernetes.Clientset, licenseSecret corev1.
 		if err != nil {
 			return appSlug, errors.Wrap(err, "failed to create airgap app")
 		}
-	} else if annotations["kots.io/airgap"] != "true" {
-		// Otherwise there is no airgap data, so this is an online install.
-		createAppOpts := online.CreateOnlineAppOpts{
-			PendingApp: &onlinetypes.PendingApp{
-				ID:           a.ID,
-				Slug:         a.Slug,
-				Name:         a.Name,
-				LicenseData:  string(license),
-				VersionLabel: instParams.AppVersionLabel,
-			},
-			UpstreamURI:            upstreamURI,
-			IsAutomated:            true,
-			SkipPreflights:         instParams.SkipPreflights,
-			SkipCompatibilityCheck: instParams.SkipCompatibilityCheck,
-		}
-		_, err := online.CreateAppFromOnline(createAppOpts)
-		if err != nil {
-			return appSlug, errors.Wrap(err, "failed to create online app")
-		}
 	}
 
 	return appSlug, nil
 }
 
-func AirgapInstall(appSlug string, additionalFiles map[string][]byte) error {
+func AutomateAirgapInstall(appSlug string, additionalFiles map[string][]byte) (finalError error) {
```

Comment:
what is the need for naming the returned parameter `finalError`?

Reply:
this method has since been refactored into the `AutomateInstall` method.  main reason for naming `finalError` was that it is referenced in the deferred function